stetho/src/main/java/com/facebook/stetho/inspector/console/CLog.java

```diff
@@ -20,7 +22,7 @@
   private static final String TAG = "CLog";
 
   public static void writeToConsole(
-      ChromePeerManager chromePeerManager,
+      @Nonnull ChromePeerManager chromePeerManager,
```

Comment:
Ok, I added the annotation because I noticed without it Intellij won't give warnings to developers while passing null for example, maybe it's worth investigating why they took this decision.